Aspire Budgeting/Views/Dashboard/DashboardCardsListView.swift

```diff
@@ -0,0 +1,61 @@
+//
+//  DashboardCardsListView.swift
+//  Aspire Budgeting
+//
+
+import SwiftUI
+
+struct DashboardCardsListView: View {
+
+  let cardViewItems: [DashboardCardView.DashboardCardItem]
+
+  var body: some View {
+    GeometryReader {g in
+      ScrollView {
+        VStack {
+          ForEach(0..<self.cardViewItems.count) { idx in
+            GeometryReader { geo in
+              BaseCardView<DashboardCardView>(
+                       minY: g.frame(in: .global).minY,
+                       curY: geo.frame(in: .global).minY,
+                baseColor: colorFor(idx: idx)) {
+                DashboardCardView(cardViewItem: self.cardViewItems[idx],
+                                  baseColor: colorFor(idx: idx))
+              }
+                .padding(.horizontal)
+            }.frame(maxWidth: .infinity)
+              .frame(height: 125)
+          }
+        }
+      }.background(Color.primaryBackgroundColor)
+    }
+  }
+}
+
+//MARK:- Internal Types
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/aspirebudgetingmobile/aspirebudgeting_ios/blob/1531d43d43f9ff835e554eb2943298d8b02e0dbb/Aspire Budgeting/Views/Dashboard/DashboardCardsListView.swift#L35"></span>MARK comment should be in valid format. e.g. ‘// MARK: …’ or ‘// MARK: - …’<br />
<code>mark</code> <code>DashboardCardsListView.swift:35</code></td>
    </tr>
  </tbody>
</table>